src/pages/Actions/ActionEvent/EventInfo/ParticipantsTable.tsx

```diff
@@ -178,78 +193,94 @@ const ParticipantsTable = ({
-    columns.push({
-      title: "Змінити статус",
-      dataIndex: "changeStatus",
-      key: "changeStatus",
-      render: (text, record) => (
-        <Space size="small">
-          {record.status != participantStatuses.Approved ? (
-            <Button
-              className="approveButton"
-              shape="round"
-              icon={<UserAddOutlined className="iconParticipant" />}
-              size="small"
-              onClick={() => {
-                changeStatusToApproved(record.participantId, record.userId);
-              }}
-            />
-          ) : (
-            <Button
-              className="disabledButton"
-              shape="round"
-              icon={<UserAddOutlined className="iconParticipant" />}
-              size="small"
-              disabled={true}
-            />
-          )}
-          <Divider type="vertical" />
-          {record.status != participantStatuses.Undetermined ? (
-            <Button
-              className="underReviewButton"
-              shape="round"
-              icon={<QuestionOutlined className="iconParticipant" />}
-              size="small"
-              onClick={() => {
-                changeStatusToUnderReviewed(
-                  record.participantId,
-                  record.userId
-                );
-              }}
-            />
-          ) : (
-            <Button
-              className="disabledButton"
-              shape="round"
-              icon={<QuestionOutlined className="iconParticipant" />}
-              size="small"
-              disabled={true}
+    columns.push(
+      {
+        title: "Відвідав подію",
+        dataIndex: "wasPresent",
+        key: "eventParticipant",
+        align: "center",
+        render: (wasPresent: boolean, record: EventParticipant) => (
+          <>
+            <Checkbox
+              onChange={() => changePresentofParticipant(record.participantId)}
+              defaultChecked={wasPresent}
             />
-          )}
-          <Divider type="vertical" />
-          {record.status != participantStatuses.Rejected ? (
-            <Button
-              className="banButton"
-              shape="round"
-              icon={<UserDeleteOutlined className="iconParticipant" />}
-              size="small"
-              onClick={() => {
-                showRejectModal(record.participantId, record.userId);
-                setRender(true);
-              }}
-            />
-          ) : (
-            <Button
-              className="disabledButton"
-              shape="round"
-              icon={<UserDeleteOutlined className="iconParticipant" />}
-              size="small"
-              disabled={true}
-            />
-          )}
-        </Space>
-      ),
-    });
+          </>
+        ),
+      },
+      {
+        title: "Змінити статус",
+        dataIndex: "changeStatus",
+        key: "changeStatus",
+        render: (text, record) => (
```

Comment:
Do you need this `text` input parameter?

Reply:
actually don't,  i have deleted this.